wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/markup/html/form/upload/progressbar.js

```diff
@@ -117,8 +135,14 @@
 			this.iframe = null;
 
 			if (progressPercent === '100') {
-				Wicket.$(this.statusid).setAttribute('hidden', '');
-				Wicket.$(this.barid).setAttribute('hidden', '');
+				var $statusId = Wicket.$(this.statusid);
+				if ($statusId != null) {
+					$statusId.setAttribute('hidden', '');
```

Comment:
Why use `hidden` attribute ?
Can we use Wicket.DOM's `hide()` and `show()` instead ?

Reply:
Maybe. I need to check. I just modified this because we had some client side errors in some conner cases in our app.

Reply:
Done